src/main/java/gov/nih/nci/evs/api/fhir/R5/FhirTerminologyCapabilitiesR5.java

```diff
@@ -0,0 +1,79 @@
+package gov.nih.nci.evs.api.fhir.R5;
+
+import ca.uhn.fhir.model.api.annotation.ChildOrder;
+import ca.uhn.fhir.model.api.annotation.ResourceDef;
+import java.util.Collections;
+import org.hl7.fhir.instance.model.api.IBaseConformance;
+import org.hl7.fhir.r5.model.ContactDetail;
+import org.hl7.fhir.r5.model.ContactPoint;
+import org.hl7.fhir.r5.model.Enumerations;
+import org.hl7.fhir.r5.model.TerminologyCapabilities;
+
+/** Specification of the FHIR TerminologyCapabilities */
+@ResourceDef(
+    name = "TerminologyCapabilities",
+    profile = "http://hl7.org/fhir/StructureDefinition/TerminologyCapabilities")
+@ChildOrder(
+    names = {
+      "url",
+      "version",
+      "name",
+      "title",
+      "status",
+      "experimental",
+      "date",
+      "publisher",
+      "contact",
+      "description",
+      "useContext",
+      "jurisdiction",
+      "purpose",
+      "copyright",
+      "kind",
+      "software",
+      "implementation",
+      "lockedDate",
+      "codeSystem",
+      "expansion",
+      "codeSearch",
+      "validateCode",
+      "translation",
+      "closure"
+    })
+public class FhirTerminologyCapabilitiesR5 extends TerminologyCapabilities
+    implements IBaseConformance {
+  /** The Constant serialVersionUID. */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * With defaults.
+   *
+   * @return the FHIR terminology capabilities
+   */
+  public FhirTerminologyCapabilitiesR5 withDefaults() {
+    setContact();
+    setCodeSystem();
+    setName("EVSRESTAPITerminologyCapabilities");
+    setStatus(Enumerations.PublicationStatus.DRAFT);
+    setTitle("EVSRESTAPI Terminology Capability Statement");
+    setVersion(getClass().getPackage().getImplementationVersion());
+    return this;
+  }
+
+  /** Sets the code system. */
+  private void setCodeSystem() {
+    final TerminologyCapabilities.TerminologyCapabilitiesCodeSystemComponent tccsc =
+        new TerminologyCapabilities.TerminologyCapabilitiesCodeSystemComponent();
```

Comment:
Ok to use hardcoded URL here? Or should this be configurable?

Reply:
That is a good question. I copied this from R4 and didn't think about that being hardcoded since it's our url. @bcarlsenca , should we be pulling the url for line 67 below from our resources instead or hardcoding this to use the evsrest url?